lib/init/helptext.html

```diff
@@ -97,4 +97,4 @@ <h2>See also</h2>
 </em>
 
 <p>
- <a href="https://epsg.io/">List of EPSG codes</a> (Database of worldwide coordinate systems)
+ <a href="https://spatialreference.org/">List of EPSG codes</a> (Database of worldwide coordinate systems)
```

Comment:
Is epsg.io that bad we can't even have it in a _See also_ section? Even the linked issue https://github.com/maptiler/epsg.io/issues/171 says:

> When presenting objects like CRS, it is probably OK, ...

CRS is what we link to even in the GUI.

Reply:
Maybe https://crs-explorer.proj.org/

Reply:
https://spatialreference.org/ is open source, now maintained by the PROJ team (currently me). It is updated on every PROJ release, every 3 months. It was completely refactored last December.

https://crs-explorer.proj.org/ is also under the PROJ umbrella (also me ;) It is older than the refactorization of SR, but SR is better known because started a long time ago (2007).

A few weeks ago SR was improved with the main functionality of crs-explorer: more complex filter, including the map.

Reply:
100% agreeing with @jjimenezshaw . Also to mention that spatialreference.org comes with no ads and no tracking, contrary to epsg.io   spatialreference.org also covers "odd" use cases such as IAU planetary CRS.  Basically it is the web view of PROJ database.

Reply:
Thank you @jjimenezshaw and @rouault. That is the info I was asking for!

Reply:
Extending my original comment, should the "See also" section link also https://crs-explorer.proj.org/ and/or https://epsg.org and/or https://epsg.io/?

Reply:
(Partially) done in 78d8a8f685b8629b3944b1ca89e9f98cab1d1032